.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     # 3. Commit this change to the PR branch where the changes exist.
     current_golden_images_hash:
         type: string
-        default: 2b6914161171d5a49857cebd254a9fbca3ed0ca4
+        default: ed603e48e9dbd4f54ad74266ee543e66ce96d7b5
```

Comment:
Can we revert the hash so we can review the changes?

Reply:
No problem, just pushed that up. We reviewed the diffs in CSS and validated those updates were as we'd expect and with no noticeable impact to customers.